NitroxServer/Communication/NetworkingLayer/NitroxServer.cs

```diff
@@ -67,5 +67,32 @@ protected void ProcessIncomingData(NitroxConnection connection, Packet packet)
                 Log.Error("Exception while processing packet: " + packet + " " + ex);
             }
         }
+
+        protected void SetupUPNP()
+        {
+            NatUtility.DeviceFound += DeviceFound;
+            NatUtility.StartDiscovery();
+        }
+
+        private async void DeviceFound(object sender, DeviceEventArgs args)
+        {
+            try
+            {
+                INatDevice device = args.Device;
+                await device.CreatePortMapAsync(new Mapping(Protocol.Udp, 11000, 11000, 86400, "Nitrox Server - Subnautica"));
```

Comment:
```suggestion
               await device.CreatePortMapAsync(new Mapping(Protocol.Udp, 11000, 11000, (int)TimeSpan.FromDays(1).TotalSeconds, "Nitrox Server - Subnautica"));
```

![image](https://user-images.githubusercontent.com/1107063/102927605-92e28000-4497-11eb-914a-d761bb4782ed.png)